sdcm/mgmt.py

```diff
@@ -4,11 +4,234 @@
 import time
 import datetime
 
+from sdcm import wait
+
 logger = logging.getLogger(__name__)
 
 STATUS_DONE = 'done'
 STATUS_ERROR = 'error'
+MANAGER_IDENTITY_FILE = '/tmp/scylla_manager_pem'
+
+from enum import Enum
+
+class TaskStatus(Enum):
+   NEW = "NEW"
+   RUNNING = "RUNNING"
+   DONE = "DONE"
+   UNKNOWN = "UNKNOWN"
+   ERROR = "ERROR"
+   STOPPED = "STOPPED"
+
+
+
+   @staticmethod
```

Comment:
Better to use `@classmethod` and `getattr` instead of lots of if-s

Reply:
but i have to convert string to enum somehow..

in python-3 it just works fine doing casting of string to enum. in python-2 i think not.

Reply:
As I wrote before you can use `getattr`[1] and `@classmethod` [2]
[1] https://docs.python.org/2/library/functions.html#getattr
[2] https://docs.python.org/2/library/functions.html#classmethod